### DISCUSSION TITLE
Hardening Extension Variant

### DISCUSSION THREAD

**Discussion by ghost** [ungoogled-software/ungoogled-chromium-android#85] [Show and tell]
Since Extension Varian Is Discontinued Maybe Some Of You Will Choose To Stay On This Version 
Outdated Browser Will Cause Some Problem In The Future

Here My Solution Is Hardening The Uc Extension:
1. Disable All Java Script And Cookies (Included First Party Cookies)
2. Instal Extension:
-Duck Duck Go Privacy
-Privacy Badger Eff.org
-NoScript Extension
-Https Everywhere
(Allow All Of These Extension In Incognito Tab And All URL)

note:
if needed just disable webview 

also Privacy browser currently develop a new webview that you can change it WITHOUT ROOTING  Your Device You May Replace Them If You Like

Best Regards
NST_Adventure

**Comment by wchen342** [comment]
This will still be a large amount of work. As I have stated, if anyone is interested in maintaining such a version it is welcomed, but I will not do it myself.

**Reply by ghost** [reply]
yes i understand sir


i just want to tell how to hardening discontinued extension version
